.travis.yml

```diff
@@ -18,16 +18,32 @@ env:
   - TOXENV=py27-dj17-cms31-postgres
   - TOXENV=flake8
 
+ global:
```

Comment:
did you use the correct credentials here? you need to create new ones with a new username on saucelabs and code climate